config/networkpolicies/90-default-deny-all.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all
```

Comment:
Same here, this policy should be a part of the bundle as this should be deployed by OLM not operator.